pvnet/models/base_model.py

```diff
@@ -582,7 +570,7 @@ def _training_accumulate_log(self, batch, batch_idx, losses, y_hat):
     def training_step(self, batch, batch_idx):
         """Run training step"""
         y_hat = self(batch)
-        y = batch[self._target_key][:, -self.forecast_len :, 0]
+        y = batch[self._target_key][:, -self.forecast_len :]
```

Comment:
Could you remind me what the shape change is here for the target? A comment could be helpful here on the shape

Reply:
Well its just that the new y values from data-sampler are only of shape [batch, time]. Previously they were [batch, time, 1] which seemed a little untidy